sql/moz-fx-data-shared-prod/firefox_desktop/newtab_visits_daily/view.sql

```diff
@@ -1,31 +1,57 @@
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_desktop.newtab_visits_daily`
 AS
+WITH src AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_derived.newtab_visits_daily_v2`
+),
+enriched AS (
+  SELECT
+    src.*,
+    -- Precompute once for reuse
+    mozfun.newtab.determine_grid_layout_v1(is_section, app_version, experiments) AS layout_type,
+    (
+      is_search_issued
+      OR is_content_interaction
+      OR is_topsite_interaction
+      OR is_widget_interaction
+      OR is_wallpaper_interaction
+      OR is_other_interaction
+    ) AS is_any_interaction,
+    (
+      is_content_interaction
+      OR is_topsite_interaction
+      OR is_widget_interaction
+      OR is_wallpaper_interaction
+      OR is_other_interaction
+    ) AS is_nonsearch_interaction,
+    (
+      is_content_interaction
+      AND NOT is_sponsored_content_interaction
+    ) AS is_organic_content_interaction,
+    (
+      is_topsite_interaction
+      AND NOT is_sponsored_topsite_interaction
```

Comment:
`is_sponsored_content_interaction` and `is_sponsored_topsite_interaction` will be true if there are _any_ sponsored content/topsite interactions, so this will only be true if the content/topsite are _exclusively_ organic. I think it might be better to write it so it will be true if there is _any_ organic content interaction but I'd like to hear from @ilanasegall on this

Reply:
The logic you suggested for the has_* in your last comment makes sense, and I think it's a valid point that it's a simple enough computation that we could remove them.  I agree with Ilana that I'd like to see the is_* columns renamed so I'd propose one of two options:

- Remove the is_* columns altogether and replace the logic dependent on them in the rollups with `LOGICAL_OR(*_count>0)`
- replace the is_* with the has_* columns as you defined them

This could be a "fast-follow" in a different PR given you'd need to make a v3 table

Reply:
@jaredsnyder @ilanasegall 
As you both confirm the change to remove (or replace) the `is_*` fields is final, if the approach is to fast-follow in a different PR where we remove the `is_*` fields, then we would need to proceed with this PR with the existing implementation of the `is_*` fields.
If we agree to this approach, we can move forward quickly on this PR by ignoring the `is_*` fields for now

Reply:
Fine by me as long as we make a ticket now so this doesn't get forgotten.  @gkatre would you mind creating a new ticket in an appropriate epic/etc, and I can fill out the details?

Reply:
That all sounds good!

For this PR, given that we want these columns to be different anyway, maybe we just remove all the new `is_*` columns from this PR?

Reply:
Sounds good